### ISSUE TITLE
Error when accessing comment in CP

### ISSUE THREAD

**Issue by ZapaNOR** [verbb/comments#255]
### Describe the bug

When trying to view a spesiffic comment in the control panel i get this error:

`Call to a member function setScenario() on null`
![Screenshot 2022-08-15 at 15 28 27](https://user-images.githubusercontent.com/53942798/184644234-d9ae59fa-9e18-432e-a48a-9c7fed6146de.png)

### Steps to reproduce

1. Instal Craft CMS 4.2.1.1
2. Install Comments '2.0.0'
3. Create a comment on an entry
4. Try to access that comment in the CP

### Craft CMS version

4.2.1.1

### Plugin version

2.0.0

### Multi-site?

yes

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/comments` requirement in `composer.json` to:

```
"require": {
  "verbb/comments": "dev-craft-4 as 2.0.0",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.1](https://github.com/verbb/comments/releases/tag/2.0.1)
